src/server.py

```diff
@@ -148,6 +156,38 @@ def binify(bins, data):
     return result
 
 
+def valve(head_rev, pushlog_id, branch, priority):
+    """Determine which kind of job should been returned"""
+    priority = priority
+    BRANCH_COUNTER.increase_the_counter(branch)
+    request_list = []
+    try:
+        request_list = session.query(TaskRequests.head_rev, TaskRequests.pushlog_id,
+                                     TaskRequests.priority).limit(40)
```

Comment:
we make it 40 because I remember you said we need to do so in https://github.com/mozilla/ouija/pull/120#discussion_r76230891